browsergym/webarena/src/browsergym/webarena/instance.py

```diff
@@ -50,6 +53,29 @@ def __init__(
 
         self.credentials = ACCOUNTS
 
+    def full_reset(self):
```

Comment:
helper function instead of code duplicate?

Reply:
What do you mean code duplicate? This code is new

Reply:
it's 90% the same code on the side of visualwebarena